src/main/java/com/exadel/discount/model/Category.java

```diff
@@ -0,0 +1,38 @@
+package com.exadel.discount.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name = "categories")
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
```

Comment:
I'm not sure, but maybe GenerationType.IDENTITY will be better?

Reply:
https://easyjava.ru/data/jpa/pervichnye-klyuchi-v-jpa/
We convented postgres as bd.

Reply:
And type should  be UUID...